org.ardour.Ardour5.json

```diff
@@ -0,0 +1,452 @@
+      ],
+      "sources" : [
+        {
+          "type" : "archive",
+          "url" : "https://downloads.sourceforge.net/project/boost/boost/1.60.0/boost_1_60_0.tar.bz2",
+          "sha256" : "686affff989ac2488f79a97b9479efb9f2abae035b5ed4d8226de6857933fd3b"
+        }
+      ]
+    },
+    {
+      "name": "liblo",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "http://download.sf.net/sourceforge/liblo/liblo-0.29.tar.gz",
+          "sha256": "ace1b4e234091425c150261d1ca7070cece48ee3c228a5612d048116d864c06a"
+        }
+      ]
+    },
+    {
+      "name": "taglib",
+      "buildsystem": "cmake-ninja",
+      "config-opts": [
+        "-DBUILD_SHARED_LIBS=ON"
+      ],
+      "cleanup": [
+        "/include",
+        "/lib/*.so"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "http://taglib.github.io/releases/taglib-1.11.1.tar.gz",
+          "sha256": "b6d1a5a610aae6ff39d93de5efd0fdc787aa9e9dc1e7026fa4c961b26563526b"
+        }
+      ]
+    },
+    {
+      "name": "vamp-plugin-sdk",
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/c4dm/vamp-plugin-sdk",
+          "tag": "vamp-plugin-sdk-v2.7.1",
+          "commit": "10be490c9902d874e9d772d94bf91448303d9b06"
+        }
+      ],
+      "post-install": [
+        "install -Dm644 -t /app/share/licenses/vamp-plugin-sdk COPYING"
+      ],
+      "cleanup": [
+        "/bin"
+      ]
+    },
+    {
+      "name": "rubberband",
+      "build-options": {
+        "env": {
+          "Vamp_CFLAGS": " ",
+          "Vamp_LIBS": " "
+        }
+      },
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://bitbucket.org/breakfastquay/rubberband/get/v1.8.1.tar.gz",
+          "sha256": "9bea85fc2e4d8c3044e3b87c6ecff4282a61a888bafe0ce07cee11fc0ea68f16"
+        }
+      ],
+      "post-install": [
+        "install -Dm644 -t /app/share/licenses/rubberband COPYING"
+      ],
+      "cleanup": [
+        "/bin",
+        "/share/ladspa"
+      ],
+      "modules": [
+        {
+          "name": "ladspa",
+          "no-autogen": true,
+          "subdir": "src",
+          "make-install-args": [
+            "INSTALL_PLUGINS_DIR=/app/lib/ladspa",
+            "INSTALL_INCLUDE_DIR=/app/include",
+            "INSTALL_BINARY_DIR=/app/bin"
+          ],
+          "cleanup": [
+            "/bin"
+          ],
+          "sources": [
+            {
+              "type": "archive",
+              "url": "http://www.ladspa.org/download/ladspa_sdk_1.15.tgz",
+              "sha256": "4229959b09d20c88c8c86f4aa76427843011705df22d9c28b38359fd1829fded"
+            }
+          ]
+        },
+        {
+          / *Double precision build */
+          "name": "fftw3",
```

Comment:
This is already in the runtime, you only need the float variant.

Reply:
Removed. All good.